ext/nnstreamer/tensor_filter/tensor_filter_llama2.cc

```diff
@@ -165,17 +182,32 @@ TensorFilterLlama2c::invoke (const GstTensorMemory *input, GstTensorMemory *outp
 
   result = get_tokens (transformer, tokenizer, sampler, prompt, steps);
 
-  /** @todo Make out_info flexible */
-  output[0].size = tokenizer->max_token_length * steps;
-  output[0].data = allocate (output[0].size);
+  if (!result) {
```

Comment:
LINE 168-181 == strndup

Reply:
LINE 174 implies that the input tensor[0] is a char string ending with '0'. Is this a safe assumption?

Reply:
Used `strndup` for input and output data :)
'\0' is arbitrarily added at LINE 171 to use `strlen`.